test/unit/puppetlabs/services/master/master_core_test.clj

```diff
@@ -84,148 +87,153 @@
-                                    "default_literal" "'literal'",
-                                    "default_source" "literal"},
-                                   {"name" "twoparam",
-                                    "type" "Integer",
-                                    "default_literal" "3",
-                                    "default_source" "3"}]
-                         },
-                        {"name" "twoclass"
-                         "params" []}]),
-                      "/two/file" (map-with-classes [])})
-               (etag {"/one/file"
-                      (map-with-classes
-                       [
-                        {"name" "oneclass",
-                         "params" [
-                                   {"default_source" "literal"
-                                    "type" "String",
-                                    "name" "oneparam",
-                                    "default_literal" "'literal'"},
-                                   {"name" "twoparam",
-                                    "type" "Integer",
-                                    "default_literal" "3",
-                                    "default_source" "3"}]
-                         },
-                        {"name" "twoclass"
-                         "params" []}]),
-                      "/two/file" (map-with-classes [])}))
-            "hashes unexpectedly not equal for equal maps")
-        (is (= (etag {"/one/file"
-                      (map-with-classes
-                       [
-                        {"name" "oneclass",
-                         "params" [
-                                   {"name" "oneparam",
-                                    "type" "String",
-                                    "default_literal" "'literal'",
-                                    "default_source" "literal"},
-                                   {"name" "twoparam",
-                                    "type" "Integer",
-                                    "default_literal" "3",
-                                    "default_source" "3"}]
-                         },
-                        {"name" "twoclass"
-                         "params" []}]),
-                      "/two/file" (map-with-classes [])})
+    (with-redefs [jruby-core/borrow-from-pool-with-timeout (fn [_ _ _] {:jruby-puppet (Object.)})
+                  jruby-core/return-to-pool (fn [_ _ _] #())]
+      (let [jruby-service (reify jruby/JRubyPuppetService
+                            #_(borrow-instance [_ _]
+                              {:jruby-puppet (Object.)})
+                            #_(return-instance [_ _ _])
+                            (get-pool-context [_] (jruby-core/create-pool-context
+                                                   (jruby-core/initialize-config {:gem-home "bar"
+                                                                                  :ruby-load-path ["foo"]})))
```

Comment:
Oops, I think this is my fault, but I think that the commented out `borrow-instance` and `return-instance` should be deleted, and also that `get-pool-context` here might be able to be deleted too.